src/card_searcher.rs

```diff
@@ -71,13 +73,35 @@ impl Search for &str {
 /// in a scryfall search parameters. The valid parameters can be seen
 /// [here](https://scryfall.com/docs/syntax).
 pub trait Param: fmt::Debug {
+    /// Adds a parameter's string version to the passed string
+    fn to_param(&self, f: &mut String);
```

Comment:
nit: This function should probably be renamed to indicate it appends this param to the string. `append_param` or `add_param` perhaps?

Reply:
It really should